site/content/docs/manifest/worker-service.en.md

```diff
@@ -134,6 +134,40 @@ Retention specifies the time a message will remain in the queue before being del
 <span class="parent-field">subscribe.queue.</span><a id="subscribe-queue-timeout" href="#subscribe-queue-timeout" class="field">`timeout`</a> <span class="type">Duration</span>  
 Timeout defines the length of time a message is unavailable after being delivered. Default 30s. Range 0s-12h.
 
+<span class="parent-field">subscribe.queue.</span><a id="subscribe-queue-fifo" href="#subscribe-queue-fifo" class="field">`fifo`</a> <span class="type">Boolean or Map</span>  
+The fifo defines the SQS FIFO Queue configurations for your default queue.   
+If you specify true, Copilot will create SQS FIFO Queue the default FIFO settings.
+
+```yaml
+subscribe:
+  topics:
+    - name: events
+      service: api
+      queue: # Define a topic-specific queue for the api-events topic.
+        timeout: 20s
+    - name: events
+      service: fe
+  queue: # By default, messages from all topics will go to a shared queue.
+    fifo: true
```

Comment:
nit: Maybe to reduce slightly the noise and keep it about FIFO:
```suggestion
subscribe:
  topics:
    - name: events
      service: api
    - name: events
      service: fe
  queue: # Messages from both FIFO SNS Topics go to the shared FIFO SQS Queue.
    fifo: true
```
When the queue is enabled with FIFO capabilities, Copilot understands that the source SNS topics must also be FIFO.